package.json

```diff
@@ -5,8 +5,9 @@
   "repository": "https://github.com/dsmHack/pi-515",
   "license": "UNLICENSED",
   "dependencies": {
-    "codemirror": "^5.46.0",
     "bulma": "^0.8.0",
+    "codemirror": "^5.46.0",
+    "firebase": "^7.10.0",
```

Comment:
If we want to do something like store secrets/keys we could use firebase. I set up a pi515 project in Nancy's firebase console account.